### PR TITLE
Update: Removed unused buttons. Log out function fails upon button click

### PR DESCRIPTION
**Pull request by ErinClaudio**
### Describe your change in plain English.

- Removed extraneous dropdown options, retaining only the "Log Out" button.
- Updated the "Log Out" button to successfully log out the user.
- Adjusted system tests to account for dynamic organization slugs and improved test reliability.

### Link to the issue
[#195](https://github.com/rubyforgood/pet-rescue/issues/195)

### Concerns
I have reservations about the sad path test for the "non-authenticated user attempts to log out" scenario. While the test passes, I believe it can be enhanced. Would appreciate guidance or a review on this particular test.

### CODE REVIEW

**Comment by kasugaijin** [comment]
Could you check to see if your change is causing the one test in the pipeline to fail?
Thanks!

**Comment by edwinthinks** [comment]
Looks like the failed test has nothing to do with the changes. 
```
LoginTest#test_: when logging in as a staff member should direct to the organization's adoptable pets page. :
  Ferrum::ProcessTimeoutError: Browser did not produce websocket url within 5 seconds, try to increase `:process_timeout`. See https://github.com/rubycdp/ferrum#customization
```

This is usually caused because the CI machine is too slow. We can fix this by making a tweak in our settings. @ErinClaudio @kasugaijin we can safetly just ignore that. I'll open a issue to resolve it.

**Comment by kasugaijin** [comment]
> Looks like the failed test has nothing to do with the changes.
> 
> ```
> LoginTest#test_: when logging in as a staff member should direct to the organization's adoptable pets page. :
>   Ferrum::ProcessTimeoutError: Browser did not produce websocket url within 5 seconds, try to increase `:process_timeout`. See https://github.com/rubycdp/ferrum#customization
> ```
> 
> This is usually caused because the CI machine is too slow. We can fix this by making a tweak in our settings. @ErinClaudio @kasugaijin we can safetly just ignore that. I'll open a issue to resolve it.

Ah yes. This is out flaky test. It failed a couple times ealier this evening too.

**Comment by ErinClaudio** [comment]
@kasugaijin @edwinthinks Thank you, for the update!

File: app/views/layouts/dashboard.html.erb [hunk 1]
```diff
@@ -197,56 +197,9 @@
...
-                    </ul>
-                    <div class="dropdown-divider"></div>
-                    <ul class="list-unstyled">
-                      <li>
-                        <a class="dropdown-item" href="../../index.html">
-                          <i class="fe fe-power me-2"></i> Sign Out
-                        </a>
-                      </li>
+                        <li>
+                          <%= button_to "<i class='fe fe-power me-2'></i> Sign Out".html_safe, destroy_user_session_path, method: :delete, class: 'dropdown-item', form: { data: { turbo: false } } %>
```
Comment (edwinthinks) [inline app/views/layouts/dashboard.html.erb:201]: To avoid having to use `.html_safe` here you could use the button_to and a pass it a block. 
```
<%= button_to url, ... do %>
   <i class='fe fe-power me-2'></i> Sign Out
<% end %>
```
You'll need to double check the syntax.

**Review by edwinthinks — CHANGES_REQUESTED** [review]
Thanks @ErinClaudio for your PR. I left you a comment let me know if you have any Qs

File: app/views/layouts/dashboard.html.erb (hunk 1 above)
Comment (ErinClaudio) [reply app/views/layouts/dashboard.html.erb:202]: Sorry, I did not mean to assign this for review. 
But thank I will make that change

Comment (ErinClaudio) [reply app/views/layouts/dashboard.html.erb]: This has been changed

**Review by edwinthinks — APPROVED** [review]
Thanks @ErinClaudio ! LGTM!
